app/packages/looker-3d/src/state.ts

```diff
@@ -127,3 +127,15 @@ export const activeNodeAtom = atom<FoSceneNode>({
   key: "fo3d-activeNode",
   default: null,
 });
+
+export const cuboidLabelLineWidthAtom = atom<number>({
+  key: "fo3d-cuboidLabelLineWidth",
+  default: 3,
+  effects: [getBrowserStorageEffectForKey("fo3d-cuboidLabelLineWidth")],
```

Comment:
I think `valueClass` needs to be added when not a string
```suggestion
  effects: [getBrowserStorageEffectForKey("fo3d-cuboidLabelLineWidth", { valueClass: "number" })],
```